x-pack/plugins/uptime/server/lib/adapters/pings/elasticsearch_pings_adapter.ts

```diff
@@ -117,13 +106,16 @@ export class ElasticsearchPingsAdapter implements UMPingsAdapter {
                   },
                 },
               },
+              filter.length ? [...filter] : undefined,
```

Comment:
Does this work when `undefined` gets passed?

Reply:
Resolved in https://github.com/elastic/kibana/pull/29374/commits/dd2916d7e8c843c50c3f67c0fffca575be496028.